tests/test_flake8_trio.py

```diff
@@ -319,13 +317,14 @@ def _parse_eval_file(test: str, content: str) -> tuple[list[Error], list[str]]:
                 msg = f'Line {lineno}: Failed to format\n {message!r}\n"with\n{args}'
                 raise ParseError(msg) from e
 
-    assert visitor_codes_regex, "no visitors enabled"
     for error in expected:
-        assert re.match(
-            visitor_codes_regex, error.code
-        ), "Expected error code not enabled"
+        for code in enable_codes.split(","):
+            if error.code.startswith(code):
```

Comment:
I'm concerned that a repeated-comma typo would enable everything here, might be worth checking that the code starts with `TRIO`?

Reply:
got confused for a moment that this was talking about the user-supplied string, but added tests that it won't break for users, and improved logic in the test so it'll warn about bad codes.